src/main/java/io/stargate/sgv2/jsonapi/api/model/command/CommandName.java

```diff
@@ -96,4 +96,8 @@ public static List<CommandName> filterByTarget(CommandTarget target) {
         .filter(commandName -> commandName.getTargets().contains(target))
         .collect(Collectors.toList());
   }
+
+  public interface Names {
```

Comment:
Don't think we need this interface we can just use the apiName property of the enum.